packages/es-components/src/components/patterns/datepicker/DatePicker.js

```diff
@@ -49,6 +61,15 @@ const NativeDatePicker = props => {
   );
 };
 
+const ReactDatePickerWrapper = ({ selectedDate, children, ...props }) => {
```

Comment:
Can we get `DatePicker`, `ReactDatePickerWrapper` and `NativeDatePicker` rewritten to use `function`? This would make it more consistent with other components in the project.

 Can we also rename `ReactDatePickerWrapper` and/or `DatePickerWrapper`? At this point `(Whatever)DatePickerWrapper` is getting fairly vague as to what the purpose of the wrapper is for.

Reply:
I renamed the general wrapper and realized I really didn't need the `ReactDatePickerWrapper` anymore. I've also updated the syntax for the NativeDatePicker to use `function`.

Reply:
Love the new name, much clearer! And removing the `ReactDatePickerWrapper` cleaned things up nicely!